cmd/kubeapps-apis/proto/kubeappsapis/plugins/fluxv2/packages/v1alpha1/fluxv2.proto

```diff
@@ -100,4 +101,23 @@ service FluxV2RepositoriesService {
       get: "/plugins/fluxv2/packages/v1alpha1/repositories"
     };
   }
+
+  rpc UpdatePackageRepository(kubeappsapis.core.packages.v1alpha1.UpdatePackageRepositoryRequest) returns (kubeappsapis.core.packages.v1alpha1.UpdatePackageRepositoryResponse) {
+    option (google.api.http) = {
+      put: "/plugins/fluxv2/packages/v1alpha1/repositories/plugin/{package_repo_ref.plugin.name}/{package_repo_ref.plugin.version}/c/{package_repo_ref.context.cluster}/ns/{package_repo_ref.context.namespace}/{package_repo_ref.identifier=**}"
+      body: "*"
+     };
+  }
+
+  // this endpoint only exists for the purpose of integration tests
```

Comment:
Not sure about having an actual extra endpoint for the integration tests?

Reply:
Well, I had a problem. I have a bunch of integration tests for AddRepository and GetRepositoryDetail. Once I split the server-side functionality of user-managed secrets and kubeapps-managed secrets a number of tests became invalid. For example, I had a test that would GetRepositoryDetail and return detail with secret refs. That only works with user-managed secrets. kubeapps-managed secrets would return Username/Password, etc. So I wanted the flexibility of switching from user-managed to kubeapps-managed and back while the server is running so that I can have a new test that returns Username/Password and the existing test that returns secretRefs. All while the server is running. My only way to do that is via a gRPC call. That's why the endpoint. The endpoint is only in the flux-specific API, BTW, not in the core API. In the future we may change how the flag works, so I may get rid of the endpoint

Reply:
Yeah, fine for now. If we need this kind of thing longer term, we could probably have a test plugin for example.

Reply:
👍